doc/api/util.md

```diff
@@ -862,6 +862,35 @@ will throw an error. If `original` is a function but its last argument is not
 an error-first callback, it will still be passed an error-first
 callback as its last argument.
 
+Due to the way JavaScript Context works using `promisify()` on class methods
```

Comment:
I don't think the phrase "JavaScript Context" is well defined

Reply:
```suggestion
Using `promisify()` on class methods
```

Reply:
@devsnek yeah, unfortunately, I couldn't find the universal way of referring to it besides just 'using `this`'.
I think @Trott's suggestion would be good.